sdks/java/extensions/avro/build.gradle

```diff
@@ -72,6 +72,7 @@ dependencies {
   }
   testImplementation library.java.avro_tests
   testImplementation library.java.junit
+  testImplementation "org.tukaani:xz:1.9" // marked as optional in avro
```

Comment:
would you mind explaining a little bit how this change needed

Reply:
so `AvroSourceTest` will fail with `java.lang.NoClassDefFoundError: org/tukaani/xz/FilterOptions` - the test is testing (among others) compression with the `XZCodec` and this requires the `org.tukaani:xz` dependency to be on the classpath - in the previous avro version this was pulled in transitively, but later it became an optional dependency that you need to add to your classpath explicilty yourself